tasks/setup-debian-pve-variant.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Copyright (C) 2018-2020 Robert Wimmer
+# Copyright (C) 2019-2020 Ties de Kock
+# Copyright (C) 2021 Steve Fan
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: (Debian) Add WireGuard repository on buster derived Proxmox
```

Comment:
but the problem is, what do we see Proxmox as? An independent OS or just a parasite of Debian?

Reply:
This is like asking if Arch and Manjaro are different, and it seems like this is quite controversial

Reply:
For me it's a different distribution because as figured out it doesn't work with the Debian tasks :wink: So we need to deal differently with it. I haven't looked at it that deeply but personally it would make most sense to handle the case here as it is the main switch point: https://github.com/githubixx/ansible-role-wireguard/blob/master/tasks/main.yml#L8-L15 But if the only difference is to figure out what distribution that is by considering the kernel name then there are not that many choices and it makes sense to handle this special case in the Debian task.